src/shared/views/SubAppsView.tsx

```diff
@@ -1,7 +1,6 @@
 import { Location } from 'history';
```

Comment:
I can't remember where we decided to leave comments from tests, so I'm leaving them here.

When the user is not logged in and they visit http://localhost:8000/search?layout=Neuron%20Electrophysiology or http://localhost:8000/my-data the screen looks a bit weird because the top of it is not fully visible:
![Screenshot from 2023-06-27 13-28-49](https://github.com/BlueBrain/nexus-web/assets/11242410/82cf01c3-3a06-4329-8618-c93f84514b0c)

![Screenshot from 2023-06-27 13-27-50](https://github.com/BlueBrain/nexus-web/assets/11242410/eec7ad28-b6d1-4fbe-892f-a6cfc4235f46)

Reply:
![Uploading Screenshot from 2023-06-27 13-28-49.png…]()